plugins/uefi-dbx/fu-uefi-dbx-common.c

```diff
@@ -29,13 +29,86 @@ fu_uefi_dbx_get_authenticode_hash(const gchar *fn, GError **error)
 	return g_strdup(fu_efi_image_get_checksum(img));
 }
 
+static GPtrArray *
+fu_uefi_dbx_get_basenames_bootxxxx(void)
+{
+	g_autoptr(GPtrArray) files = g_ptr_array_new_with_free_func(g_free);
+
+	/* hardcoded, as we chainload from shim/grub to these */
+	g_ptr_array_add(files, g_strdup("fwupdx64.efi"));
+	g_ptr_array_add(files, g_strdup("fwupdaa64.efi"));
+	g_ptr_array_add(files, g_strdup("fwupdia32.efi"));
```

Comment:
we know the arch we're running on via `EFI_MACHINE_TYPE_NAME`, use that to build the string.

Reply:
Fixed.